src/main/java/com/klarna/hiverunner/builder/HiveShellBase.java

```diff
@@ -100,7 +102,16 @@
     }
```

Comment:
Why can't all command shell specific transformation happen in the transformScript call. The returned sql would have all the imports/!run commands etc. exploded and ready to be executed by the regular hive parser. This way it's easy to add more of the beeline commands etc without leaking outside of the emulator